libs/nox/array/README.md

```diff
@@ -0,0 +1,55 @@
+# nox/array
```

Comment:
What to call this? Its path is "nox/array". Its package is "nox-array". Its crate is "nox_array". I suggest we call it "nox_array" to match its crate name.